docs/devguide/modules/install-sbo/partials/openshift/external-cr/helm-install.adoc

```diff
@@ -0,0 +1,41 @@
+=== Install with Helm
+
+1. Build and push the operator's image
++
+[source,bash]
+----
+export OPERATOR_IMAGE_REF=$REGISTRY/service-binding-operator:$(git rev-parse --short=8 HEAD)
+make image push-image -o registry-login
+----
+
+2. Install the CRDs
++
+[source,bash]
+----
+make install
+----
+
+3. Deploy the operator
++
+[source,bash]
+----
+helm install \
+    --create-namespace \
+    --namespace openshift-operators \
+    --set image.repository=$OPERATOR_IMAGE_REF \
```

Comment:
See the https://github.com/redhat-developer/service-binding-operator/pull/1309#discussion_r1030395699